generators/csharp/codegen/src/project/CsharpProject.ts

```diff
@@ -609,7 +609,7 @@ ${this.getAdditionalItemGroups().join(`\n${FOUR_SPACES}`)}
         return result;
     }
 
-    private relativePathToWindowsPath(relativePath: RelativeFilePath): RelativeFilePath {
-        return RelativeFilePath.of(path.win32.normalize(relativePath));
+    private relativePathToWindowsPath(relativePath: RelativeFilePath): string {
+        return path.win32.normalize(relativePath);
```

Comment:
Note that this change was required due to the changes we introduced in https://github.com/fern-api/fern/pull/4674.

That PR is working just as intended - we just shouldn't have been using `RelativeFilePath` in this case.